_posts/2022-11-21-november-demo-day.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: Dask Demo Day November 2022
+author: Richard Pelgrim (Coiled)
+tagline: an optional tagline
+tags: [Community]
+theme: twitter
+draft: true
+---
+
+{% include JB/setup %}
+
+# Dask Demo Day : November
+
+Once a month, the Dask Community team hosts Dask Demo Day: an informal and fun online hangout where folks can showcase new or lesser-known Dask features…and the rest of us can learn about all the things we didn’t know Dask could do 😁
+
+November’s Dask Demo Day had 5 great demo's. We learned about:
+- the Dask-Optuna integration for distributed hyperparameter optimization
+- visualizing 2-billion lightning flashes with Dask, RAPIDS and Datashader
+- the new Dask CLI
+- Dask-Awkward
+- profiling your Dask code with Dask-PySpy 
+
+This blog gives you a quick overview of the 5 demos and demonstrates how they might be useful to you. You can watch the full recording below.
```

Comment:
```suggestion
This blog gives you a quick overview of the 5 demos and demonstrates how they might be useful to you. You can [watch the full recording below](https://www.youtube.com/embed/_x7oaSEJDjA").
```

Reply:
I'd recommend providing an additional link here to the YouTube video, because I've noticed that sometimes embedded videos do not display correctly on previous dask blog pages [like this one](https://blog.dask.org/2021/06/25/dask-down-under). Giving a backup link makes that problem less of an issue if & when that happens, since they can just click to open it in another tab.

(I'm not sure why I see this problem happen in the first place, might be the adblocking extensions I have, or the browser, or maybe even with the connection to youtube since the error message shown is "www.youtube.com refused to connect")